projects/js-packages/publicize-components/src/components/social-previews/modal.scss

```diff
@@ -15,6 +18,13 @@
 	}
 }
 
+.jetpack-social-previews__modal--close-btn {
+	position: absolute;
+    right: 0.5rem;
```

Comment:
is this misaligned?

Reply:
That means sass formatting is not properly set up in the repo. It's tabs vs spaces there. I have fixed it. Thanks for noticing that.